### ISSUE TITLE
Results are not reproducible on MIMIC-Ⅲ

### ISSUE THREAD

**Issue by ChongWang2000** [reml-lab/mTAN#3]
I use the data extraction process to get the MIMIC-Ⅲ dataset which contains 53,211 records. I use the given code to split the train, valid and test set. And I use the hyperparameters given to run mTAN-full, but I do not achieve 0.8544 AUROC on MIMIC-Ⅲ dataset . The highest is AUROC ~0.838.

This is achieved with this command:
python3 tan_classification.py --alpha 5 --niters 300 --lr 0.0001 --batch-size 128 --rec-hidden 256 --gen-hidden 50 --latent-dim 128 --enc mtan_rnn --dec mtan_rnn   --save 1 --classif --norm --learn-emb --k-iwae 1 --dataset mimiciii

[Classification Task on MIMIC-III Dataset (mTAND-Full).log](https://github.com/reml-lab/mTAN/files/7559622/Classification.Task.on.MIMIC-III.Dataset.mTAND-Full.log)

**Comment by wang67-jianli** [comment]
Hi! How to get this: No such file or directory: '../../../neuraltimeseries/Dataset/final_input3.npy'?
